style3.css

```diff
@@ -62,28 +65,17 @@ ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 1043px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
 	z-index: 5;
 }
-
-ul li #a{
```

Comment:
27/09/2021: I was making a mistake. because I was making a mistake on setting left margin for the drop down menu column. So the I need to create two difference drop down menu column. But I have found out and fix the issue so I don't need this anymore.